typescript/packages/browser-lite/src/index.ts

```diff
@@ -0,0 +1,20 @@
+export * from './mesh';
+
+// Start compiling the WebAssembly as soon as the script is loaded. This lets
+// us initialize as quickly as possible.
+const go = new Go();
+WebAssembly.instantiateStreaming(fetch('main.wasm'), go.importObject)
```

Comment:
I think requiring users to serve the WebAssembly bytecode at `"main.wasm"` is too strict. It prevents people from doing things like pulling in the bytecode from IPFS or using a CDN that is separate from their main URL. Can we re-architect this so that you pass in the URL or Promise in order to get the bytecode?